media-video/vdr/files/vdr-2.2.0_gentoo.patch

```diff
@@ -1,7 +1,6 @@
-diff -Naur vdr-2.1.7.orig/Makefile vdr-2.1.7/Makefile
---- vdr-2.1.7.orig/Makefile	2015-01-18 12:37:45.506034673 +0100
-+++ vdr-2.1.7/Makefile	2015-01-18 12:38:34.086037162 +0100
-@@ -116,7 +116,7 @@
+--- a/Makefile	2019-11-10 12:45:03.370809670 +0100
++++ b/Makefile	2019-11-10 12:48:55.100815904 +0100
```

Comment:
if we're already invalidating the distfiles cache, might as well clean it up properly:
```
--- a/Makefile
+++ b/Makefile
```

Reply:
lines truncated as requested...